src/components/scrollToTop.js

```diff
@@ -0,0 +1,17 @@
+import MenuIcon5 from "../assets/menu-icons-05.png";
+
+const ScrollToTop = () => {
```

Comment:
This sounds like a function, not like a component. I think we could rename this. If the first word in a name is an action, it is always a function usually. Plus the file name for components need to be capital letter